prisma/migrations/20220925203103_init/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- CreateEnum
+CREATE TYPE "Role" AS ENUM ('FACULTY', 'MERIT_COMMITTEE_MEMBER', 'MERIT_COMMITTEE_HEAD', 'DEAN');
+
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "email" TEXT NOT NULL,
```

Comment:
Shouldn't this be a unique field as 2 users usually cannot have the same email?

Reply:
seems like this is something that prisma handles